Example/Tests/GiniVisionTestsHelper.swift

```diff
@@ -1,50 +1,47 @@
 //
-//  Extensions.swift
+//  GiniVisionTestsHelper.swift
 //  GiniVision_Tests
 //
-//  Created by Enrique del Pozo Gómez on 9/11/17.
-//  Copyright © 2017 Gini GmbH. All rights reserved.
+//  Created by Enrique del Pozo Gómez on 6/5/18.
+//  Copyright © 2018 Gini GmbH. All rights reserved.
 //
 
 import Foundation
-import UIKit
-import XCTest
 @testable import GiniVision
 
-internal extension XCTestCase {
```

Comment:
Why can't this still be an extension to the `XCTestCase` class? It was more convenient to call methods on self and not on a separate class. 
And how does having class methods improve performance?

Reply:
When fetching the images/pdfs from the bundle an instance of `self` had to be provided, being a problem when loading these files in a non-lazy way.
I decided to do it in a separate class because at the end, although it was more convenient and in terms of performance is the same, it is not the responsibility of the TestCase to load a file.

What improved was not the class methods but not loading the same file several times (there was a problem with `lazy`properties).

Reply:
I understand the performance improvements, but now the code is not a easy to read and write.